jujugui/static/gui/src/test/test_ghost_deployer_extension.js

```diff
@@ -157,6 +157,18 @@ describe('Ghost Deployer Extension', function() {
     assert.equal(services.item(3).get('name'), 'mysql-a');
   });
 
+  it('increments the name with deleted services', function() {
```

Comment:
This test should probably be expanded to make sure that it's not actually renaming existing services